src/mmw/js/src/account/views.js

```diff
@@ -13,7 +15,64 @@ var Clipboard = require('clipboard'),
     accountTmpl = require('./templates/account.html');
 
 var ProfileView = Marionette.ItemView.extend({
-    template: profileTmpl
+    template: profileTmpl,
+
+    ui: {
+        firstName: '#first_name',
+        lastName: '#last_name',
+        organization: '#organization',
+        userType: '#user_type',
+        country: '#country',
+        postalCode: '#postal_code',
+        saveChanges: '[data-action="save_changes"]'
+    },
+
+    events: {
+        'click @ui.saveChanges': 'saveChanges'
+    },
+
+    modelEvents: {
+        'change': 'render'
+    },
+
+    onRender: function() {
+        var self = this,
+            addOptions = function(model, field, selectedValue) {
+                $.each(window.clientSettings.choices[model][field], function(index, choice) {
```

Comment:
Consider prepending an unselected value, or defaulting to the US. It currently displays "Afghanistan" if you haven't given a country.

Also, consider importing [`core/settings`](https://github.com/WikiWatershed/model-my-watershed/blob/2dc52e9b704dee29a29401d290b31a29c04e0251/src/mmw/js/src/draw/controllers.js#L6) instead of pulling it off the window.

Reply:
Do you have a specific set of steps that resulted in the country defaulting to `Afghanistan`? `US` is specified as the default in both the backend and frontend and when I registered a new account it did show US as the default.

https://github.com/WikiWatershed/model-my-watershed/blob/4b242006ed09df325362650218f600735aaa306f/src/mmw/apps/user/models.py#L73

https://github.com/WikiWatershed/model-my-watershed/blob/4b242006ed09df325362650218f600735aaa306f/src/mmw/js/src/user/templates/userProfileModal.html#L14

 

I will switch to an import for client setting.

Reply:
I ran the query in your testing instructions and refreshed `/account`. I think we're setting the default for the modal, but not the profile page:

https://github.com/WikiWatershed/model-my-watershed/blob/db7e61d8960b60596b98615d857447418a2edcdc/src/mmw/js/src/account/templates/profile.html#L55

Reply:
Thanks. I'll dig into that.

Reply:
Addressed in fixup 098ea6f

Reply:
This still uses `window.clientSettings`, but I think that's okay.

Reply:
Sorry. Lost commit. Actually fixed in the last rebase.